spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -35,17 +63,19 @@ def and_i_upload_a_file_attachment
     click_on "Save and continue"
   end
 
-  def then_i_can_see_previews_of_the_attachment
-    metadata = "PDF, 13 KB, 1 page"
+  def then_i_can_see_the_attachment
+    @metadata = "PDF, 13 KB, 1 page"
 
     within(".gem-c-attachment") do
       expect(page).to have_content(@title)
-      expect(page).to have_content(metadata)
+      expect(page).to have_content(@metadata)
     end
+  end
 
+  def and_i_can_see_previews_of_the_attachment
```

Comment:
I think we should be more accurate with the naming here. This is only checking a single type of preview now: the inline version. I think it's important we try and keep the original meaning of the step, so maybe use a different step for each scenario, or rename this one to be about the inline preview specifically.

Reply:
So, I need to revert this line, because the preview does work for both inline and featured attachments, however the test fails. So I need to investigate a bit more.

Reply:
I'm getting my previews confused.